0 silly input [ 'start' ]
1 silly flags { _: [ 'run' ],
1 silly flags   'reject-cycles': false,
1 silly flags   rejectCycles: false,
1 silly flags   parallel: true,
1 silly flags   script: 'start',
1 silly flags   args: [] }
2 verbose rootPath /Users/jonatanpedersen/jonatanpedersen/jambon
3 info version 2.5.1
4 silly existsSync /Users/jonatanpedersen/jonatanpedersen/jambon/VERSION
5 silly initialize attempt
6 silly initialize success
7 silly execute attempt
8 info run in 1 package(s): npm run start
9 silly runScriptInPackageStreaming [ 'start', [], 'foo-service' ]
10 silly getExecOpts { cwd: '/Users/jonatanpedersen/jonatanpedersen/jambon/services/foo-service' }
11 error execute callback with error
12 error Error: Command failed: npm run start
12 error events.js:183
12 error       throw er; // Unhandled 'error' event
12 error       ^
12 error
12 error Error: listen EADDRINUSE :::8000
12 error     at Object._errnoException (util.js:1024:11)
12 error     at _exceptionWithHostPort (util.js:1046:20)
12 error     at Server.setupListenHandle [as _listen2] (net.js:1351:14)
12 error     at listenInCluster (net.js:1392:12)
12 error     at Server.listen (net.js:1476:7)
12 error     at main (/Users/jonatanpedersen/jonatanpedersen/jambon/services/foo-service/lib/main.js:58:9)
12 error     at <anonymous>
12 error     at process._tickCallback (internal/process/next_tick.js:188:7)
12 error npm ERR! code ELIFECYCLE
12 error npm ERR! errno 1
12 error npm ERR! foo-service@1.0.0 start: `node .`
12 error npm ERR! Exit status 1
12 error npm ERR!
12 error npm ERR! Failed at the foo-service@1.0.0 start script.
12 error npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
12 error
12 error npm ERR! A complete log of this run can be found in:
12 error npm ERR!     /Users/jonatanpedersen/.npm/_logs/2017-11-22T21_02_03_544Z-debug.log
12 error
12 error
12 error > foo-service@1.0.0 start /Users/jonatanpedersen/jonatanpedersen/jambon/services/foo-service
12 error > node .
12 error
12 error
12 error     at Promise.all.then.arr (/Users/jonatanpedersen/jonatanpedersen/jambon/node_modules/lerna/node_modules/execa/index.js:236:11)
12 error     at <anonymous>
